### ISSUE TITLE
NavigationActions have lost their `toString()` implementations

### ISSUE THREAD

**Issue by jamesreggio** [react-navigation/react-navigation#4072]
@ericvicenti, in #3619 you deleted the tests that confirm that each `NavigationAction` function has a `toString()` that returns its action name.

I had added these back in #2260, and the justifications remain. I'm going to hack them back in for my own usage, and I'd be happy to submit a PR if this was merely an oversight.

If you have philosophical disagreements with them, let's discuss — but, at minimum, you should note it as a breaking change and remove it from the documentation [here](https://v2.reactnavigation.org/docs/en/navigation-actions.html) (search for `toString()`).

### Current Behavior

```js
import {NavigationActions} from 'react-navigation';
NavigationActions.navigate.toString() !== 'Navigation/NAVIGATE';
```

### Expected Behavior

```js
NavigationActions.navigate.toString() === 'Navigation/NAVIGATE';
```

...and so on, for each action creator.

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 2.0.0-rc.4

**Comment by ericvicenti** [comment]
Is this toString behavior in the docs? If so, let’s put it back. But honestly I’d rather just keep the actions simple and report this as a breaking change.

**Comment by jamesreggio** [comment]
Yeah, the `toString()` behavior is in the docs. It's still a common pattern to match against the action-creator function with various Redux libraries.

I'll send you a PR tomorrow.
